tests/TestCase.php

```diff
@@ -24,6 +24,8 @@ public function getModelWithOptions($options)
         return new class($options) extends Model {
             use SearchString;
 
+            protected $table = 'dummyModel';
+
```

Comment:
Not very important but I'm just curious to understand why this needed to be added? 🙂

Reply:
This as added dude to somekind of test failure running PHP 7.4.3 before i even began adding this feature. I cannot remember excatly what the issue was. But it was something regarding asserting the actual query in the test when we dumped the sql using a closure yeilded another table name string each time which the regex did not find.